cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js

```diff
@@ -68,8 +68,8 @@ describe('ui-data-import: A user can filter and delete import logs from the "Vie
     Users.deleteViaApi(secondUser.userId);
     // TODO delete all created instances and holdings
   });
-
-  it('C358136 A user can filter and delete import logs from the "View all" page (folijet)', { tags: [TestTypes.smoke, DevTeams.folijet] }, () => {
+  // removed tag smoke because test runs infinite
```

Comment:
locally this test passed. I can say that it is the issue with cloud resources. I have other issues like this. I think that this happens because of limitation of resources in the cloud

Reply:
I notify Tania about this comment, when we will not have infinite test in pipeline we uncomment this. right now it affect whole test run